specific/cmo_sale/models/sale.py

```diff
@@ -93,11 +98,11 @@ def _compute_project_number(self):
 
     @api.model
     def _default_covenant(self):
-        convenants = self.env['sale.convenant.description'].search([
+        covenants = self.env['sale.covenant.description'].search([
             ['active', '=', True],
         ])
-        if convenants:
-            return convenants[0].description
+        if covenants:
+            return _(covenants[0].description)
```

Comment:
You shouldn't put a variable inside _(...), it will try to translate it.
The description, should already have translation.